spec/classes/periodicnoise_spec.rb

```diff
@@ -20,4 +20,20 @@
       .with_content(%r{somehost.example.com}) \
       .with_content(%r{monitor.example.com})
   end
+
+  it 'should be configured globally to send messages to one host' do
+    should contain_file('/etc/periodicnoise/config.ini')\
+      .with_content(Regexp.new Regexp.escape 'OK       = printf "somehost.example.com;%(event);0;%(message)\n" |/usr/sbin/send_nsca -H monitor.example.com -d ";"')
```

Comment:
I thought that `Regexp.escape` returns a regexp object, but it turns out it doesn't. So ignore that comment.

Please still use brackets, e.g. `Regexp.new(Regexp.escape(..))`.